ci/environment-3.7.yml

```diff
@@ -4,7 +4,7 @@ channels:
 dependencies:
   - python=3.7
   - xarray
-  - dask
+  - dask[array]
```

Comment:
Ah, I see what the issue might be now. `dask[array]` (and my comment here https://github.com/xgcm/xhistogram/pull/71#pullrequestreview-810615021) is just a `pip` thing and isn't supported when installing `dask` with `conda`. `conda install dask` will pull in other packages like `bokeh` though. Specifically the packages listed [here](https://github.com/conda-forge/dask-feedstock/blob/98bc1b513a8df48ef1cf5c10f1d20eea61ea9691/recipe/meta.yaml#L20-L27). 

If you just want to `conda install` the minimum required dependencies for Dask (which would be equivalent to `pip install dask`), you'll want to install the `dask-core` package from `conda-forge`.

Reply:
using `dask-core` reduces `Setup Conda Env` from 5:37 https://github.com/xgcm/xhistogram/runs/4258344230?check_suite_focus=true to 2:47 https://github.com/xgcm/xhistogram/runs/4258767295?check_suite_focus=true